ReactAndroid/src/main/java/com/facebook/react/views/image/ReactImageView.java

```diff
@@ -82,13 +90,29 @@ float getRadius(Bitmap source) {
             0.0f,
             mScaleType);
         sMatrix.invert(sInverse);
-        return sInverse.mapRadius(mBorderRadius);
+
+        mappedRadii[0] = sInverse.mapRadius(computedCornerRadii[0]);
+        mappedRadii[1] = mappedRadii[0];
+
+        mappedRadii[2] = sInverse.mapRadius(computedCornerRadii[1]);
+        mappedRadii[3] = mappedRadii[2];
+
+        mappedRadii[4] = sInverse.mapRadius(computedCornerRadii[2]);
+        mappedRadii[5] = mappedRadii[4];
+
+        mappedRadii[6] = sInverse.mapRadius(computedCornerRadii[3]);
+        mappedRadii[7] = mappedRadii[6];
     }
 
     @Override
     public void process(Bitmap output, Bitmap source) {
+      cornerRadii(sComputedCornerRadii);
+
       output.setHasAlpha(true);
-      if (mBorderRadius < 0.01f) {
+      if (sComputedCornerRadii[0] < EPS &&
```

Comment:
lets use `FloatUtil.floatsEqual(sComputedCornerRadii[0], 0f)` here instead so that we don't need to have epsilon constants scattered around the codebase in many different places